ramalama/model.py

```diff
@@ -774,5 +774,6 @@ def compute_serving_port(port: str, debug: bool) -> str:
 
     if target_port == 0:
         raise IOError("no available port could be detected. Please ensure you have enough free ports.")
-    print(f"serving on port {target_port}")
+    if not quiet:
+        print(f"serving on port {target_port}")
```

Comment:
You could simplify this and delete this completely if you wanted. llama-server and vllm tend to display the IP address and port anyway